Automattic/wp-calypso
# chore(@automattic/composite-checkout): Upate webpack version

#### Changes proposed in this Pull Request

* Delete unused webpack.config.js
* Update webpack to 5.13.0

#### Testing instructions

Run `yarn composite-checkout-demo` in the root, verify the composite checkout demo page loads at http://localhost:3000
